README.adoc

```diff
@@ -439,7 +459,7 @@ Next, run the following command to view your newly created `OpenLibertyApplicati
 oc get OpenLibertyApplications
 ```
 
-You can also replace `OpenLibertyApplications` with the shortname `olapps`.
+You can also replace `OpenLibertyApplications` with its shortname `olapps`.
```

Comment:
avoid showing possession by inanimate objects- 

You can also replace `OpenLibertyApplications` with the shortname `olapps`.